### PR TITLE
Prepare to merge the CLIs

### PR DESCRIPTION
**Pull request by doylemark**
# Why
Prepare to merge the main evervault cli

resolving ETR-1732

# How
Move all enclave commands to their own directory. Nothing changes here except moving all the modules in src into ./src/enclaves and updating all the inputs.

Since everything moved into `enclaves` dir, renamed to the existing `enclaves` modules to `nitro`, because the nested enclave mods made imports very hard to read (and update).

Had to resolve all the imports by hand 😩

### CODE REVIEW

File: crates/ev-enclave/src/enclave/api/assets.rs [hunk 1]
```diff
@@ -1,6 +1,5 @@
 use super::client::{ApiClient, ApiClientError, ApiResult, GenericApiClient, HandleResponse};
-use crate::api::client::ApiError;
-use crate::api::client::ApiErrorKind;
+use crate::enclave::{api::client::ApiError, api::client::ApiErrorKind};
```
Comment (lfarrel6) [inline crates/ev-enclave/src/enclave/api/assets.rs:2]:
```suggestion
use crate::enclave::api::client::{ApiError, ApiErrorKind};
```

File: crates/ev-enclave/src/enclave/cli/logs.rs [hunk 2]
```diff
@@ -1,10 +1,8 @@
-use crate::api;
-use crate::api::AuthMode;
-use crate::common::CliError;
-use crate::config::EnclaveConfig;
+use crate::enclave::{
+    api, api::AuthMode, common::CliError, config::EnclaveConfig, logs::get_logs,
```
Comment (lfarrel6) [inline crates/ev-enclave/src/enclave/cli/logs.rs:2]:
```suggestion
    api::{AuthMode, self}, common::CliError, config::EnclaveConfig, logs::get_logs,
```

File: crates/ev-enclave/src/enclave/cli/scale.rs [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-use crate::version::check_version;
-use crate::{
+use crate::enclave::{
+    api,
     api::{
         enclave::{EnclaveApi, EnclaveClient},
         AuthMode,
     },
+    common,
     common::CliError,
```
Comment (lfarrel6) [inline crates/ev-enclave/src/enclave/cli/scale.rs:8]:
```suggestion
    common::{CliError, self},
```

File: crates/ev-enclave/src/enclave/cli/scale.rs [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
     config::EnclaveConfig,
```
Comment (lfarrel6) [inline crates/ev-enclave/src/enclave/cli/scale.rs:9]: This can move into the import below

File: crates/ev-enclave/src/enclave/encrypt/mod.rs [hunk 5]
```diff
@@ -1,10 +1,11 @@
-use crate::config::EnclaveConfigError;
-use crate::{
+use crate::enclave::{
+    api,
     api::{
         enclave::{EnclaveApi, EnclaveClient},
         AuthMode,
     },
```
Comment (lfarrel6) [inline crates/ev-enclave/src/enclave/encrypt/mod.rs:6]:
```suggestion
    api::{
        enclave::{EnclaveApi, EnclaveClient},
        AuthMode,
        self,
    },
```

**Review by lfarrel6 — APPROVED** [review]
lgtm, just a few import nits

**Comment by doylemark** [comment]
closing for now in favour of #106

**Pull request closed without merging**